### PR TITLE
Only forbid resizing when videorecorder is recording

### PR DESCRIPTION
**Pull request by def-**

### CODE REVIEW

File: src/engine/client/video.h [hunk 1]
```diff
@@ -60,6 +60,7 @@ class CVideo : public IVideo
 	virtual void start();
 	virtual void stop();
 	virtual void pause(bool p);
+	virtual bool isRecording() { return m_Recording; }
```
Comment (heinrich5991) [inline src/engine/client/video.h:63]: Naming convention.

Comment (def-) [reply src/engine/client/video.h]: Done.

**Comment by heinrich5991** [comment]
bors r+

Thanks for the cleanup. :)

**Comment by sirius1242** [comment]
@def- This caused SIGFPE when rendering video in my machine, revert 83c820d solved it. Does someone else have this problem?

**Comment by def-** [comment]
I'll take a look, sorry about that.

**Comment by def-** [comment]
can reproduce.

**Comment by def-** [comment]
Fixed in https://github.com/ddnet/ddnet/pull/2461

**Comment by sirius1242** [comment]
It worked, Thanks!

**Pull request merged**